benchmark/src/main/java/ai/timefold/solver/benchmark/impl/aggregator/swingui/BenchmarkAggregatorFrame.java

```diff
@@ -131,8 +130,7 @@ public BenchmarkAggregatorFrame(BenchmarkAggregator benchmarkAggregator) {
         this.benchmarkAggregator = benchmarkAggregator;
         benchmarkResultIO = new BenchmarkResultIO();
         plannerBenchmarkResultList = Collections.emptyList();
-        Locale locale = benchmarkAggregator.getBenchmarkReportConfig().determineLocale();
-        millisecondsSpentNumberFormat = new MillisecondsSpentNumberFormat(locale);
+        millisecondsSpentNumberFormat = new MillisecondsSpentNumberFormat();
```

Comment:
A) Either the benchmark report has a locale.
B) Or the locale is the eye of the beholder. Basically, the browser's locale determines if it's  1'000,05 (BE) or 1,000.05 (US).

If B) is possible, I prefer that. A big advantage over the JfreeChart approach is that the locale isn't hard coded in the report output.

Anyway:
- In A), the MillisecondsSpentNumberFormat should still use the locale. Even if it's just for tables in html.
- In B), the millisecondsSpentNumberFormat can be removed, as no number is formatted during report generation (only during report rendering).